### PR TITLE
Added unit test

### PR DESCRIPTION
**Pull request by ThanyaniDev**
Unit Test added

### CODE REVIEW

File: MarsMissionTests/ForecastRepositoryImplemantationTest.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		
+		XCTAssertEqual("2020-11-05T22:00:00.000+0000",forecastResults?.forecasts[0].date)
+		XCTAssertEqual(40.0,forecastResults?.forecasts[1].temp)
+		XCTAssertEqual(30,forecastResults?.forecasts[2].humidity)
+		XCTAssertEqual(6000,forecastResults?.forecasts[3].windSpeed)
+		XCTAssertEqual(false,forecastResults?.forecasts[4].safe)
+		XCTAssertEqual("NASA Mars North Weather Station",forecastResults?.weatherStation)
+		XCTAssertEqual("2020-11-07T22:00:00.000+0000",forecastResults?.lastUpdated)
+    }
+}
```
Comment (LehlohonoloIsaac) [inline MarsMissionTests/ForecastRepositoryImplemantationTest.swift:42]: Shouldn't the failure case be tested as well?

Comment (ThanyaniDev) [reply MarsMissionTests/ForecastRepositoryImplemantationTest.swift:42]: Screenshot 2020-11-19 at 19 21 51

File: MarsMissionTests/ForecastRepositoryImplemantationTest.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		})
+		
+		XCTAssertEqual(nil,forecastResults?.forecasts[0].date)
+		XCTAssertEqual(nil,forecastResults?.forecasts[1].temp)
+		XCTAssertEqual(nil,forecastResults?.forecasts[2].humidity)
+		XCTAssertEqual(nil,forecastResults?.forecasts[3].windSpeed)
+		XCTAssertEqual(nil,forecastResults?.forecasts[4].safe)
+		XCTAssertEqual(nil,forecastResults?.weatherStation)
+		XCTAssertEqual(nil,forecastResults?.lastUpdated)
+		XCTAssertNil(forecastResults)
```
Comment (LehlohonoloIsaac) [inline MarsMissionTests/ForecastRepositoryImplemantationTest.swift]: It's sufficient to only assert that `forecastResults ` is nil as in line 58: `XCTAssertNil(forecastResults)`

Comment (ThanyaniDev) [reply MarsMissionTests/ForecastRepositoryImplemantationTest.swift]: Screenshot 2020-11-20 at 12 04 26

**Pull request merged**